index.html

```diff
@@ -125,9 +127,13 @@ <h1>Identifying AV1 streams in MPEG-2 TS</h1>
 
 ### Semantics
 
+**descriptor_tag** - This value shall be set to TBD (pick in H.222 user private range).
```

Comment:
Can we replace the TBD before merging? Also why do we mention H.222 and not MPEG-2 TS?

Reply:
My understanding is that we have to use a value in the MPEG User-defined range (0x40-0xFF), but it has to be also in the DVB User Range, so in 0x80-0xFE. AVS3 uses 209 (0xD1).

Reply:
Fix in f34d1770af58b39e976942d8b1f2152d943c5a76 I took 101

Reply:
I actually picked 101 in the 0x40-0xFF, I'm not sure why we need to be in the DVB user range since this is just embedded in the payload of the private data specifier ? I can change to another value, what do you think ?

Reply:
Fix in 3f5fa3d8a9a7915e5ba785d0c1e5b562eaf367fe